tests/phpunit/Persistence/SitelinkBasedStatementsLookupTest.php

```diff
@@ -162,4 +162,26 @@ public function testPageWithMultipleSitelinksToItemsWithStatementsReturnsStateme
 		);
 	}
 
+	public function testCustomNamespacePageWithSitelinkToItemWithStatementsReturnsStatements(): void {
+		$item = new Item(
+			id: new ItemId( 'Q1' ),
+			statements: new StatementList(
+				new Statement( new PropertyValueSnak( new NumericPropertyId( 'P1' ), new StringValue( 'foo' ) ) ),
+				new Statement( new PropertyValueSnak( new NumericPropertyId( 'P2' ), new StringValue( 'bar' ) ) )
+			)
+		);
+		$page = $this->createPage( namespace: 1337 );
```

Comment:
Why this namespace? Does it exist when the tests are run?

Reply:
It didn't exist, but was enough to trigger the title difference. It is not created correctly in https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/243